.circleci/config.yml

```diff
@@ -18,67 +7,60 @@ jobs:
-          name: Install core utilities
-          command: make install
-      - run:
-          name: Setup the Virtual Environment
+          name: Setup the Virtual Environment and install dependencies
           command: make venv
-      - restore_cache:
-          name: Restore pip cache
-          keys:
-            - &pip_cache_key venv-x-{{ checksum "requirements.txt" }}
-      - run:
-          name: Install dependencies
-          command: pipenv run -- make deps
-      - save_cache:
-          name: Save pip cache
-          key: *pip_cache_key
-          paths:
-            - venv
       - run:
           name: Run linting
-          command: pipenv run -- make lint
+          command: make lint
   unit_tests:
+    docker:
+        - image: 'circleci/python:3.7'
+    steps:
+        - checkout
+        - run:
+            name: Setup the Virtual Environment and install dependencies
+            command: make venv
+        - run:
+            name: Install dependencies
+            command: make deps
+        - run:
+            name: Run unit tests
+            command: make test
+  deploy:
     docker:
       - image: 'circleci/python:3.7'
     steps:
       - checkout
       - run:
-          name: Install core utilities
-          command: make install
+          name: Copy in aws config
+          command: |
+            mkdir -p ~/.aws
+            cp .circleci/aws_config ~/.aws/config
       - run:
-          name: Setup the Virtual Environment
+          name: Setup the Virtual Environment and install dependencies
           command: make venv
-      - restore_cache:
-          name: Restore pip cache
-          keys:
-            - *pip_cache_key
       - run:
-          name: Install dependencies
-          command: pipenv run -- make deps
-      - save_cache:
-          name: Save pip cache
-          key: *pip_cache_key
-          paths:
-            - venv
+          name: upload to internal security
+          command:  |
+            AWS_ACCESS_KEY_ID=$INTERNAL_ACCESS_KEY_ID \
+            AWS_SECRET_ACCESS_KEY=$INTERNAL_SECRET_ACCESS_KEY \
+            AWS_DEFAULT_REGION=$INTERNAL_DEFAULT_REGION \
+            pipenv run -- panther_analysis_tool upload --filter Tags=internal --aws-profile internal
       - run:
-          name: Run unit tests
-          command: pipenv run -- make test
-  schema_tests:
-      docker:
-        - image: 'circleci/python:3.7'
-      steps:
-        - checkout
-        - download_pantherlog_tool:
-            version: v1.17.0-RC
-        - run:
-            name: Run schema tests
-            command: ./pantherlog -v && ./pantherlog test ./schemas || true
+          name: upload to hosted security
+          command: |
+            AWS_ACCESS_KEY_ID=$HOSTED_ACCESS_KEY_ID \
+            AWS_SECRET_ACCESS_KEY=$HOSTED_SECRET_ACCESS_KEY \
+            AWS_DEFAULT_REGION=$HOSTED_DEFAULT_REGION \
+            pipenv run -- panther_analysis_tool upload --filter Tags=hosted --aws-profile hosted
```

Comment:
❓

I think this was meant for the dogfood repo

Reply:
Yeah this is my mistake. I made a bad assumption that the CI job was the same for both, I'll be committing an updated Pipfile shortly